apps/dashboard/src/lib/components/CourseContainer/index.svelte

```diff
@@ -29,6 +29,7 @@
     const { data: _data } = await fetchCourse(courseId);
 
     if (_data) {
+      $course.course_type = _data.course_type;
```

Comment:
i didn't even think of that initially, but looking at it now that logic isn't needed because the setcourse function already sets the local course store to the data being passed in